services/apps/src/apps/clients/permissions.clj

```diff
@@ -140,3 +140,19 @@
 (def unshare-app (partial unshare-resource (rt-app)))
 (def share-analysis (partial share-resource (rt-analysis)))
 (def unshare-analysis (partial unshare-resource (rt-analysis)))
+
+(defn- get-public-resource-ids [resource-type]
+  (->> (pc/get-subject-permissions-for-resource-type (client) "group" (ipg/grouper-user-group-id) resource-type false)
```

Comment:
This works by setting the `lookup` flag to `false` in the call to `get-subject-permissions-for-resource-type`. Having the `lookup` flag set to `false` means that only permissions that are assigned directly to the subject will be listed. [To be honest, this is the only available permission search for groups. I thought it was best to have the lookup type explicitly set to `false` anyway.]